packages/app/src/contacts/components/sync-contacts-modal/input-file-select.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { SyncContactsModalTestIds } from "App/contacts/components/sync-contacts-modal/sync-contacts-modal-test-ids.enum"
+
+export interface Props {
+  fileInputRef: React.RefObject<HTMLInputElement>
+  accept?: string
+  onManualImportClick: (inputElement: HTMLInputElement) => void
+  onCancelManualImportClick: () => void
+}
+
+const InputFileSelect = ({
+  accept = ".vcf",
+  fileInputRef,
+  onCancelManualImportClick,
+}: Props): JSX.Element => {
+  const handleFocusBack = () => {
+    onCancelManualImportClick()
+    window.removeEventListener("focus", handleFocusBack)
+  }
+
+  return (
+    <input
+      ref={fileInputRef}
```

Comment:
[forwardRef](https://react.dev/reference/react/forwardRef) seams more suitable for this case